core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -51,18 +53,54 @@ angular.module('oppia').directive('imageUploader', [
             return 'This file is not recognized as an image.';
           }
 
-          if (!file.type.match('image/jpeg') &&
-            !file.type.match('image/gif') &&
-            !file.type.match('image/jpg') &&
-            !file.type.match('image/png') &&
-            !file.type.match('image/svg\\+xml')) {
+          var imageTypeMapping = {
```

Comment:
Just to check, have you tested the image-uploader directive with images of different formats listed here?

Reply:
I didn't check for all image types before but now I have checked it.